### ISSUE TITLE
Add "information on the social entities represented in OMRSE" to wiki

### ISSUE THREAD

**Issue by dexink** [mcwdsi/OMRSE#252]
The README for OMRSE states the following:

> For more information on the social entities represented in OMRSE, please visit our [wiki page](https://github.com/mcwdsi/OMRSE/wiki) or list of publications.

Currently, the Wiki is limited to two pages:

1. the landing page https://github.com/mcwdsi/OMRSE/wiki
2. an overview page https://github.com/mcwdsi/OMRSE/wiki/OMRSE-Overview

### The following are proposed objectives for the OMRSE Wiki to be supplemented in parallel to continued maintenance and releases:

1. Identify the fundamental entities and ontological components that require mention and determine the respective extent of exposition required for each
2. Define an informative, intuitive structure by which to organize the OMRSE Wiki and "social entities" represented therein

**Secondary Objective:**
- [For each entity in Wiki] Catalog all mappings and source ontologies (if entities/classes were merged/reused); do so in a standardized, easily reproducible manner cognizant of release numbers, access dates

**Comment by hoganwr** [comment]
Not true. There are multiple pages under the modeling heading I added to the Wiki:

https://mcwdsi.github.io/OMRSE/modeling/Home/

There are some significant gaps, and major new developments that override portions of it, so work to do. But it's definitely there.

**Comment by hoganwr** [comment]
WHich also highlights that you're looking in the wrong place. We will not use the WIki on the GitHUb site going forward.

All OMRSE documentation has officially moved to here for development: https://github.com/mcwdsi/OMRSE/tree/main/docs/modeling

And published here: https://mcwdsi.github.io/OMRSE/modeling/Home/

**Comment by dexink** [comment]
So the links within the READ.ME need to be updated?

**Comment by hoganwr** [comment]
The links within the README file have been updated. Also, the main Wiki
page on the GitHub site has also been updated to point to the new location.

Note that this issue as currently written is perhaps overly broad. What
information needs to be added to which pages?  Each page update should be
specified in detail probably as its own issue (or even set of issues
depending).
